src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1315,6 +1315,7 @@
     <Content Include="sharpdx_direct3d11_effects_x86.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Resource Include="UI\Images\trash_icon.png" />
```

Comment:
hmm.. I was surprised that we did not use font awesome for this. Is this icon from UX?

Reply:
@QilongTang I removed the png image from the project and I used the FontAwesome toolit for the trash icon, see the commit: https://github.com/DynamoDS/Dynamo/pull/11636/commits/0fb11cde9886451ae02192f52dfc92552696ce67